stories/organisms/components/filePicker/FilePicker.stories.mdx

```diff
@@ -0,0 +1,164 @@
+import { ArgsTable, Meta, Source, Story, Canvas, Preview, Props } from '@storybook/addon-docs'
+import { FilePicker } from 'ui/organisms/filePicker/FilePicker'
+import { Typography } from 'ui/atoms/typography/Typography'
+import { ThemeProvider } from 'ui/atoms/theme/ThemeProvider'
+import { ThemeSwitcher } from 'ui/atoms/theme/ThemeSwitcher'
+import { StoreProvider } from 'ui/providers/storeProvider'
+import cosmosDark from 'assets/images/cosmos-dark.png'
+import cosmosLight from 'assets/images/cosmos-clear.png'
+import { useTheme } from 'hook/useTheme'
+import { default as storeParameters } from './store'
+
+<Meta
+  title="Organisms/FilePicker"
+  component={FilePicker}
+  parameters={{
+    actions: { argTypesRegex: '^on.*' },
+    docs: {
+      source: {
+        type: 'code'
+      }
+    }
+  }}
+  decorators={[
+    Story => (
+      <ThemeProvider>
+        <div className="story-theme-switcher">
+          <ThemeSwitcher />
+        </div>
+        <StoreProvider storeParameters={storeParameters}>
+          <div className="component-story-main filePicker">
+            <Story />
+          </div>
+        </StoreProvider>
+      </ThemeProvider>
+    )
+  ]}
+/>
+
+# OKP4 File Picker
+
+> Allows to select files and display them.
+
+[![stability-unstable](https://img.shields.io/badge/stability-unstable-yellow.svg)](https://github.com/emersion/stability-badges#unstable)
+
+## Description
+
+The `FilePicker` component is a component that displays a file input and the list of the selected files.  
+It is a combination of the [FileInput](/docs/atoms-fileinput--overview) component and the [List](/docs/atoms-list--overview) and [Listitem](/docs/atoms-listitem--overview) components.
```

Comment:
I agree to rewrite the description. But I think that the word "upload" is not appropriate here because the component does not allow to upload the files. 
What do you think @lolottetheclash @ccamel @fredericvilcot ?